libr/util/punycode.c

```diff
@@ -15,7 +15,7 @@ ut32 adapt_bias(ut32 delta, unsigned n_points, int is_first) {
 	delta /= is_first ? DAMP : 2;
 	delta += delta / n_points;
 
-	for (k = 0; k > ((BASE - TMIN) * TMAX) / 2; k += BASE) {
+	for (k = 0; delta > ((BASE - TMIN) * TMAX) / 2; k += BASE) {
```

Comment:
This looks hard to read/understand because k is not used in the body nor in the condition, but just later. I think a `while` would be better, with the explicit setting of k before the while and the increment in the body of the loop.